CMakeLists.txt

```diff
@@ -148,7 +148,7 @@ else (APPLE)
       set(CMAKE_EXE_LINKER_FLAGS "-Wl,--large-address-aware")
    else (MINGW)
       set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -fPIC -g")
-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -O2 -DNDEBUG -DQT_NO_DEBUG")
+      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -Os -DNDEBUG -DQT_NO_DEBUG")
```

Comment:
Optimize for size surely is good for appimages, but not really needed for 'regular' (Linux) packages?